test/appengine/test_gae_connectionpool.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+import logging
+import sys
+
+from nose.plugins.skip import SkipTest
+
+try:
+    from google.appengine.api import urlfetch
+except ImportError:
+    raise SkipTest("App Engine SDK not available.")
+
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
+
+from urllib3 import encode_multipart_formdata
+
+from urllib3.contrib.gaepool import (
+    connection_from_url,
+    GAEConnectionPool,
+)
+
+from urllib3.exceptions import (
+    ConnectTimeoutError,
+    EmptyPoolError,
+    DecodeError,
+    MaxRetryError,
+    TimeoutError,
+    ProtocolError,
+)
+
+from urllib3.packages.six import b, u
+
+
+from test.with_dummyserver.test_connectionpool import TestConnectionPool
+
+
+# Note that this doesn't run in the sandbox, it only runs with the URLFetch
+# API stub enabled. There's no need to enable the sandbox as we know for a fact
+# that URLFetch is used by the connection pool.
+class TestGAEConnectionPool(TestConnectionPool):
+    nosegae_urlfetch = True
```

Comment:
Could you note that this is a magic variable used by nosegae directly? (Probably in the class comment earlier)